core/embed/rust/src/ui/shape/render.rs

```diff
@@ -118,12 +123,14 @@ where
 // struct ProgressiveRenderer
 // ==========================================================================
 
+// #[cfg(not(feature = "xframebuffer"))]
```

Comment:
I would recommend placing `ProgressiveRenderer` in its own file, rather than decorating each detail in the current file with `#[cfg...]`.

Reply:
moved https://github.com/trezor/trezor-firmware/pull/3974/commits/4fb6e8f9b552f3fbff6e384ec8df33edd4e49793